logger/sentry.go

```diff
@@ -0,0 +1,122 @@
+package logger
+
+import (
+	"fmt"
+
+	"github.com/getsentry/sentry-go"
+)
+
+// A SentryLogger
+type SentryLogger struct {
+	l Logger
+	//ctx LogContext
+}
+
+// NewSentryLogger constructs a SentryLogger based off the provided TrailsLogger.
+func NewSentryLogger(tl *TrailsLogger, dsn string) Logger {
+	err := sentry.Init(sentry.ClientOptions{
+		Dsn:          dsn,
+		Environment:  tl.env,
+		IgnoreErrors: []string{"write: broken pipe"},
+	})
+	if err != nil {
+		err = fmt.Errorf("unable to init Sentry: %s", err)
+		tl.Error(err.Error(), nil)
+		return tl
+	}
+
+	tl.skip = 3
+	return &SentryLogger{l: tl}
+}
+
+// Debug writes a debug log.
+func (sl *SentryLogger) Debug(msg string, ctx *LogContext) {
+	sl.l.Debug(msg, ctx)
+	//	sl.l.WithContext(sl.ctx).Debug(msg)
```

Comment:
Curious why you left this in?

Reply:
For later reference should we want to update the API.

Reply:
Maybe discuss the pros/cons and then move forward with an option now? These kind of maybes rarely get revisited and they add cruft when trying to understand what does exist